airbyte-integrations/connectors/source-mixpanel-singer/sample_files/sample_config.json

```diff
@@ -4,5 +4,5 @@
   "attribution_window": "5",
   "project_timezone": "US/Pacific",
   "select_properties_by_default": "true",
-  "start_date": "2019-01-01T00:00:00Z"
+  "start_date": "2020-10-01T00:00:00Z"
```

Comment:
Why this change? If there was a corresponding change in the secrets/config.json file, make sure you update it in lastpass.

Reply:
to get rid of the warning about too old start_date (1 year is max), my initial thought was that this is the cause of broken test, but it is not